Linear/Test/TestImageReader.C

```diff
@@ -0,0 +1,63 @@
+//==============================================================================
+//!
+//! \file TestImageReader.C
+//!
+//! \date Sept 2018
+//!
+//! \author Kjetil A. Johannessen / SINTEF
+//!
+//! \brief Tests for image reader
+//!
+//==============================================================================
+
+#define STB_IMAGE_IMPLEMENTATION
+#include "stb_image.h"
+
+#include "gtest/gtest.h"
+
+
+TEST(TestImageReader, png)
+{
+  int width, height, nrChannels;
+  unsigned char *image = stbi_load("refdata/chess.png",
+                                   &width,
+                                   &height,
+                                   &nrChannels, 0);
+  ASSERT_EQ(width,      512);
+  ASSERT_EQ(height,     512);
+  ASSERT_EQ(nrChannels,   4);
```

Comment:
I would say it is better to use EXPECT_EQ instead of ASSERT_EQ in these cases where it makes sense to check all cases and not only until one fails.

Reply:
The code below (checks pixel values) assumes the dimension of the image to be 512x512 pixels and would crash if this is not the case. I think it is correct to use `ASSERT` here?

Reply:
Yes, I agree when checking sizes of arrays, etc., but not when checking the value of each array element.